easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb

```diff
@@ -0,0 +1,79 @@
+easyblock = 'PythonPackage'
+
+name = 'SCGid'
+version = '0.9b0'
+# No tagged commit for SCGid 0.9b, taking commit from commit history
+local_commit = 'be106e4cd888fd913a527ea821e510e0afb100be'
+
+homepage = 'https://github.com/amsesk/SCGid'
+description = 'A consensus approach to contig filtering and genome prediction from single-cell sequencing libraries'
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ["https://github.com/amsesk/SCGid/archive"]
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+patches = [
+    "SCGid-0.9-fixPrintToPython3.patch",
+    "SCGid-0.9b0-initEdit.patch"
+]
+checksums = [
+    '1f625e8f87b4d5e4ec9831ebe48763676679d6d6c41b05ef0bb59e6a7f839153',  # SCGid-0.9b0.tar.gz
+    'ee251b71ed3eed097b05751a3cea302d954000e1a8112e6e3f297e67c84996f7',  # SCGid-0.9-fixPrintToPython3.patch
+    '7ae6759deffbb4f4c07943fd9aa44739e5b3d0e9ef724cee3e1dc03b71ad4735',  # SCGid-0.9b0-initEdit.patch
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('R', '4.1.2'),
+    ('BLAST+', '2.12.0'),
+    ('AUGUSTUS', '3.4.0'),
+    ('Java', '11', '', True),
+    ('Maven', '3.6.3', '', True),
+    ('plotly.py', '5.4.0'),
+    ('ETE', '3.1.2'),
+    ('PyYAML', '5.4.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = False
+
+exts_defaultclass = "Tarball"
+
+options = {'modulename': False}
+
+local_clams_commit = "e326c34307803b9ea8ffc975ddd9bd05643af931"
+
+exts_list = [
+    ('ClaMS-CLI', '31012020', {
+        'source_urls': ['https://github.com/amsesk/ClaMS-CLI-fork/archive'],
+        'sources': [{
+            'download_filename': '%s.tar.gz' % local_clams_commit,
+            'filename': 'ClaMS-CLI-%(version)s.tar.gz'
+        }],
+        'checksums': ['743397fc8308e363f127bd7e5165ac9d769c10305f6e33d9e408db8f343170db'],
+    }),
+    ('ESOM', '1.1', {
+        'download_instructions': """
+         You need to manually provide Databionics ESOM. Please, follow these steps:
```

Comment:
@ItIsI-Orient Final request for change: I would tweak these instructions a bit, to make them clearer (less room for interpretation).

Done in https://github.com/ItIsI-Orient/easybuild-easyconfigs/pull/1